documentation/INSTALL.md

```diff
@@ -1,43 +1,117 @@
 # Installation of Software
```

Comment:
I think this install guide is way to compilicated.

Why not do it the same way as the build scripts. https://github.com/MusicQuizCPP/Music-Quiz_CPP/blob/master/build_scripts/dependencies_windows.sh
Basically:

1. Install choco: https://chocolatey.org/
2. Run

```
choco install NuGet.CommandLine -y
choco install 7zip -y
nuget install boost -Version 1.72.0 -OutputDirectory /c/boost

curl -LO https://code.qt.io/cgit/qbs/qbs.git/plain/scripts/install-qt.sh;
./install-qt.sh   --version 5.12.9 --toolchain win64_msvc2017_64 qtbase qtsvg qtmultimedia qttools
```

Reply:
I updated the install guide to include both options. Is that satisfactory?